### ISSUE TITLE
Password rule for zdf.de has incorrect form

### ISSUE THREAD

**Issue by Scetch** [apple/password-manager-resources#232]
The rule for `zdf.de` ends with `allowed: lower; special;`. I'm not sure if `special` was supposed to be in its own rule or `;` should simply be `,`.

```
"zdf.de": {
    "password-rules": "minlength: 8; required: upper; required: digit; allowed: lower; special;"
},
```

https://github.com/apple/password-manager-resources/blob/main/quirks/password-rules.json#L310

**Comment by DerJuulsn** [comment]
Dang! You're totally right!

**Comment by DerJuulsn** [comment]
Solved in #236
